From 657e3ab2d1aefe7cab92349b884c616b517bf200 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 30 Jul 2018 08:41:29 +0000
Subject: [PATCH] OS-7090 GDT limit reset to 0xffff Reviewed by: Robert
 Mustacchi <rm@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 kvm_vmx.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kvm_vmx.c b/kvm_vmx.c
index e585b20..ffac0c6 100644
--- a/kvm_vmx.c
+++ b/kvm_vmx.c
@@ -813,6 +813,8 @@ __vmx_load_host_state(struct vcpu_vmx *vmx)
 		rdmsrl(MSR_KERNEL_GS_BASE, vmx->msr_guest_kernel_gs_base);
 		wrmsrl(MSR_KERNEL_GS_BASE, vmx->msr_host_kernel_gs_base);
 	}
+
+	reset_gdtr_limit();
 }
 
 static void
